CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Bug fixes
+* Modifying basis handling in `skbio.stats.composition.ilr_inv` prior to checking for orthogonality.  Now the basis is strictly assumed to be in the Aitchison simplex. ([#1393](https://github.com/biocore/scikit-bio/issues/1393))
```

Comment:
Also the issue number/link is no longer relevant to this change, so you can just drop it entirely.